src/NServiceBus.Core.Tests/ApprovalFiles/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -1924,6 +1924,13 @@ namespace NServiceBus.ObjectBuilder
         public static void Release(this System.IServiceProvider serviceProvider, object instance) { }
     }
 }
+namespace NServiceBus.OpenTelemetry
+{
+    public static class OpenTelemetryConfigurationExtensions
+    {
+        public static void EnableOpenTelemetry(this NServiceBus.EndpointConfiguration endpointConfiguration) { }
```

Comment:
`EnableOpenTelemetryInstrumentation` ?

Reply:
We discussed this and decided that `EnableOpenTelemetry` is fine for an initial release.